src/_pytest/unittest.py

```diff
@@ -63,6 +62,13 @@ class UnitTestCase(Class):
     # to declare that our children do not support funcargs.
     nofuncargs = True
 
+    def newinstance(self, method_name: str = "runTest"):
+        # TestCase __init__ takes the method (test) name, so TestCaseFunction
+        # needs to pass it.
+        # runTest is a special no-op name for unittest, can be used when a dummy
+        # instance is needed.
```

Comment:
I don't quite get this comment... "name for unittest" you mean pytest's unittest, or stdlib unittest? Also is this recommending calling runTest to obtain a new dummy instance?